modules/retail/pages/retail-uyuni-server-setup.adoc

```diff
@@ -0,0 +1,292 @@
+Passwords must be at at least seven characters in length, and must not contain spaces, single or double quotation marks (``'`` or ``"``), exclamation marks (``!``), or dollar signs (``$``).
+Always store your passwords in a secure location.
++
+Click btn:[Next] to continue.
+
+. Click btn:[Yes] to begin the setup process.
+
+. When setup is complete, click btn:[Next] to continue.
+Take note of the address to access the {productname} {webui}.
+
+. Click btn:[Finish] to complete {productname} setup.
+
+
+
+== Create the Main Administration Account
+
+This section covers how to create your organization's main administration account for {productname}.
+
+[WARNING]
+====
+The main administration account has the highest authority within {productname}.
+Ensure you keep access information for this account secure.
+We recommend that you create lower level administration accounts for organizations and groups.
+Do not share the main administration access details.
+====
+
+
+.Procedure: Setting Up the Main Administration Account
+
+. In your web browser, enter the address for the {productname} {webui}.
+This address was provided after you completed setup.
+For more information, see xref:retail:retail-uyuni-server-setup.adoc#retail-server-setup-yast[].
+
+. Sign in to the {webui}, navigate to the menu:Create Organization[Organization Name] field, and enter your organization name.
+
+. In the menu:Create Organization[Desired Login] and menu:Create Organization[Desired Password] fields, enter your username and password.
+
+. Complete the ``Account Information`` fields, including an email for system notifications.
+
+. Click btn:[Create Organization] to finish creating your administration account.
+
+When you have completed the {productname} {webui} setup, you are taken to the menu:Home[Overview] page.
+
+
+
+== Add Software Channels
+
+Before you register {productname} branch servers and terminals to your {productname} Server, check that you have the {opensuse} product enabled, and the required channels are fully synchronized.
+
+The products you need for this procedure are:
+
+[[opensuse-channels-cli]]
+[cols="1,1,1,1,1", options="header"]
+.OpenSUSE Channels - CLI
+|===
+
+| OS Version
+| Base Channel
+| Client Channel
+| Updates Channel
+| Other Channels
+
+| {opensuse} Leap 15.2
+| opensuse_leap15_2
+| opensuse_leap15_2-uyuni-client
+| opensuse_leap15_2-updates
+| uyuni-proxy-stable-leap-152
+
+|===
+
+include::snippets/add_channels_cli.adoc[]
+
+
+
+== Check Synchronization Status
+
+include::snippets/check_sync_webui_uyuni.adoc[]
+
+include::snippets/check_sync_cli.adoc[]
+
+[IMPORTANT]
+====
+{opensuse} channels can be very large.
+Synchronization can sometimes take several hours.
+====
+
+
+
+== Trust GPG Keys on Clients
+
+include::snippets/trust_gpg.adoc[]
+
+
+
+
+
+// FIXME Starting from here, everything is preliminary
+// Feedback provided by Lukas
+== Create Activation Key for a Proxy (Branch Server) and the Retail Terminals
```

Comment:
Since we're in the retail book, I would call it a branch server in the title, then explain that it's a proxy in the prose following.